subcommand/acl-controller/command.go

```diff
@@ -48,16 +51,19 @@ func (c *Command) init() {
 	c.flagSet.StringVar(&c.flagPartition, flagPartition, "", "The Consul partition name that the ACL controller will use for ACL resources. If not provided will default to the `default` partition [Consul Enterprise]")
 	c.flagSet.BoolVar(&c.flagPartitionsEnabled, flagPartitionsEnabled, false, "Enables support for Consul partitions and namespaces [Consul Enterprise]")
 
-	c.log = hclog.New(nil)
+	logging.Merge(c.flagSet, c.LogOpts.Flags())
```

Comment:
Borrowed this approach from Consul: https://github.com/hashicorp/consul/blob/319fe485613bed39784252f275f221d117235b78/command/login/login.go#L64-L68